app/assets/javascripts/admin/addon/components/chart.gjs

```diff
@@ -6,11 +6,20 @@ import loadScript from "discourse/lib/load-script";
 // chartConfig - object
 export default class Chart extends Component {
   renderChart = modifier((element) => {
-    const { chartConfig } = this.args;
+    const { chartConfig, loadChartDataLabelsPlugin } = this.args;
 
-    loadScript("/javascripts/Chart.min.js").then(() => {
-      this.chart = new window.Chart(element.getContext("2d"), chartConfig);
-    });
+    loadScript("/javascripts/Chart.min.js")
+      .then(
+        () =>
+          loadChartDataLabelsPlugin &&
+          loadScript("/javascripts/chartjs-plugin-datalabels.min.js")
```

Comment:
this is a bit weird pattern wise. I wonder if they can load concurrently some how ... maybe worth asking dev-xp what the ideal pattern is? 

technically we should be able to kick off evaluating the js for both payloads at the same time... also I wonder if maybe we just have one payload for chart.js?

Reply:
Hmm the `Promise.all` won't work - https://chartjs-plugin-datalabels.netlify.app/guide/getting-started.html

> `chartjs-plugin-datalabels` must be loaded **after** the Chart.js library!

and there's no bundled `Chart.js` distribution that includes the datalabels plugin 😐